s3fs/core.py

```diff
@@ -1034,7 +1047,9 @@ async def _get_file(
         try:
             with open(lpath, "wb") as f0:
                 while True:
-                    chunk = await body.read(2**16)
+                    chunk = await _error_wrapper(
+                        body.read, args=(2**16,), retries=self.retries
```

Comment:
I'm don't think this solves the original issue. 
If the server disconnected, we cannot repeat the read on the stream as the stream has gone away. We would need to restart the transfer, except that we should start from the current size of the output file.